payee/payeeDAO_test.go

```diff
@@ -0,0 +1,80 @@
+package payee
+
+import (
+	"context"
+	"database/sql"
+	"os"
+	"testing"
+
+	_ "github.com/lib/pq"
+)
+
+func setupTestDB(t *testing.T) *sql.DB {
+	dsn := os.Getenv("TEST_DATABASE_URL")
+
+	db, err := sql.Open("postgres", dsn)
+	if err != nil {
+		t.Fatalf("failed to connect to DB: %v", err)
+	}
+	if err := db.Ping(); err != nil {
+		t.Fatalf("failed to connect to DB: %v", err)
+	}
+	return db
+}
+
+func TestInsertAndGetPayee(t *testing.T) {
+	db := setupTestDB(t)
+	defer func() {
+		if err := db.Close(); err != nil {
+			t.Errorf("failed to close DB connection: %v", err)
```

Comment:
[not-a-blocker-for-pr-merge][reading-suggestion] while we are doing the right thing by doing db.close, but log line suggests that we need to understand more about DB object, db.close is not just about closing the connection